packages/client/src/rpc/modules/debug.ts

```diff
@@ -457,4 +460,35 @@ export class Debug {
     this.client.config.logger.configure({ level: logLevels[level] })
     return `level: ${this.client.config.logger.level}`
   }
+
+  /**
+   * Sets the current head of the local chain by block number. Note, this is a
+   * destructive action and may severely damage your chain. Use with extreme
+   * caution.
```

Comment:
This looks like the docs from geth, are these also applicable here?

Reply:
Since it is implemented with `force=true`, I'd say it can be destructive, so it would be good to know in the docs.